uk.co.compucorp.civicrm.tasksassignments/CRM/Tasksassignments/Upgrader.php

```diff
@@ -920,6 +920,29 @@ public function upgrade_1037() {
     return TRUE;
   }
 
+  /**
+   * Set default number of days before document expiry when the clone document
+   * should be created = 90 days If this field is blank, update it to 90 days.
+   *
+   * @return bool
+   */
+  public function upgrade_1038() {
+    $optionValue = civicrm_api3('OptionValue', 'get', [
+      'sequential' => 1,
+      'option_group_id' => 'ta_settings',
+      'name' => 'days_to_create_a_document_clone',
+    ]);
```

Comment:
Would be good to check 
```php
if(empty($optionValue['id'])) {
return TRUE;
}
```
I know it should exist but just good to write the code in a safe way.